betty/conf/app.py

```diff
@@ -20,8 +20,8 @@ class BettyConfConfig(AppConfig):
 PACKAGE_DIR = os.path.dirname(os.path.dirname(__file__))
 
 DEFAULTS = {
-    "BETTY_IMAGE_ROOT": os.path.join(_settings.MEDIA_ROOT, "images"),
-    "BETTY_IMAGE_URL": urljoin(_settings.MEDIA_URL, "images/"),
+    "BETTY_IMAGE_ROOT": _settings.MEDIA_ROOT,
+    "BETTY_IMAGE_URL": urljoin(_settings.MEDIA_URL, "/"),
```

Comment:
Eventually NO, this will be a drop-in replacement that shouldn't break existing configs.